tests/acquisition/covidcast/test_csv_importer.py

```diff
@@ -42,6 +42,35 @@ def test_is_sane_week(self):
     self.assertFalse(CsvImporter.is_sane_week(202054))
     self.assertFalse(CsvImporter.is_sane_week(20200418))
 
+  def test_find_issue_specific_csv_files(self):
+      """Recursively explore and find issue specific CSV files."""
+      # valid path
+      path_prefix='prefix/to/the/data/issue_20200408'
+      os.makedirs(path_prefix, exist_ok=True)
+      self.assertTrue(os.path.isdir(path_prefix))
```

Comment:
see if you can do this with [`unittest.mock.patch`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch) instead -- maybe something like

```
@patch("os.path.isdir")
def test_find_issue_specific_csv_files(self, os_isdir_mock):
    ...
    os_isdir_mock.return_value = True
    ...
```